packages/core/src/customer/strategies/braintree/braintree-paypal-credit-customer-strategy.ts

```diff
@@ -141,9 +141,11 @@ export default class BraintreePaypalCreditCustomerStrategy implements CustomerSt
         let hasRenderedSmartButton = false;
 
         if (paypal) {
+            const checkoutUserExperienceSettings = this._store.getState().config.getConfig()
+                ?.storeConfig.checkoutSettings.checkoutUserExperienceSettings;
             const fundingSources = [paypal.FUNDING.PAYLATER, paypal.FUNDING.CREDIT];
             const commonButtonStyle = {
-                height: 40,
+                height: checkoutUserExperienceSettings?.walletButtonsOnTop ? 36 : 40,
```

Comment:
Could we instead pass the value (36) from Checkout JS into here? I imagine the height depends on the value we set in Checkout JS. So custom checkouts might have a different value?

Reply:
Thank you for your approval, DC.